src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -148,6 +152,34 @@ public void start(Future<Void> startFuture) throws Exception {
                 this.router.errorHandler(HttpResponseStatus.BAD_REQUEST.code(), this::errorHandler);
                 this.router.errorHandler(HttpResponseStatus.NOT_FOUND.code(), this::errorHandler);
 
+                //enable cors
+                if (this.bridgeConfig.getHttpConfig().isCorsEnabled()) {
+                    Set<String> allowedHeaders = new HashSet<>();
+                    //set predefined headers
+                    allowedHeaders.add("x-requested-with");
+                    allowedHeaders.add("Access-Control-Allow-Origin");
+                    allowedHeaders.add("Access-Control-Allow-Methods");
+                    allowedHeaders.add("origin");
+                    allowedHeaders.add("Content-Type");
+                    allowedHeaders.add("accept");
```

Comment:
Maybe there are some constants for all these headers in `io.netty.handler.codec.http.HttpHeaderNames` instead of hard-coded strings?

Reply:
yes done it with commit baba18db6e6f2a4e7391106cd27d1ca5b4f63869
not found a constant for string "x-requested-with"